app/models_types.py

```diff
@@ -0,0 +1,349 @@
+import datetime
+from typing import Any, NotRequired, TypedDict
+
+
+class SerializedUser(TypedDict):
+    id: str
```

Comment:
This is a `UUID`, same as a few other items, so to make this reflect the types of the current data we'd need the lines:

```py
id: UUID
current_session_id: UUID | None
organisations: list[UUID]
services: list[UUID] | list[dict]
```

Or alternatively we could convert them to strings in the serialize function - it looks like we already do that in other places.